src/lib.rs

```diff
@@ -137,6 +137,7 @@
 //! See our [MSRV policy].
 //!
 //! [MSRV policy]: https://github.com/google/zerocopy/blob/main/POLICIES.md#msrv
+#![cfg_attr(not(feature = "byteorder"), doc = "[`byteorder`]: #")]
 
```

Comment:
Rustfmt wants to get rid of this newline, which IMO is a bug. TODO: Accept rustfmt's opinion? Wait until this bug is fixed? Something else?

https://github.com/rust-lang/rustfmt/issues/5947

Reply:
```suggestion
// Without this comment, rustfmt incorrectly tries to get rid of the following newline. 

```

Reply:
Good idea! Done.